src/layouts/MainLayout.vue

```diff
@@ -1,25 +1,17 @@
 <template>
   <q-layout view="lHh Lpr lFf">
-    <q-header elevated>
-      <q-toolbar>
-        <q-btn
-          flat
-          dense
-          round
-          icon="menu"
-          aria-label="Menu"
-          @click="toggleLeftDrawer"
-        />
-
-        <q-toolbar-title>
-          Quasar App
+    <q-header class="bg-white text-grey-10" bordered>
+      <q-toolbar class="constrain">
+        <q-btn to="/camera" class="large-screen-only q-mr-sm" flat round icon="eva-camera-outline" size="20px" dense />
+        <q-separator class="large-screen-only" vertical spaced/>
+        <q-toolbar-title class="text-grand-hotel text-bold">
+          Quasagram
         </q-toolbar-title>
-
-        <div>Quasar v{{ $q.version }}</div>
+        <q-btn to="/" class="large-screen-only" flat round icon="eva-home-outline" size="20px" dense />
       </q-toolbar>
     </q-header>
 
-    <q-footer class="bg-white" bordered>
+    <q-footer class="bg-white small-screen-only" bordered>
            <q-tabs class=" text-grey-10" active-color="primary" indicator-color="transparent">
             <q-route-tab  to="/" icon="eva-home-outline" />
             <q-route-tab  to="/camera" icon="eva-camera-outline"/>
```

Comment:
![image](https://user-images.githubusercontent.com/59572810/219298379-df402e0e-e657-4204-b886-9d755e903c28.png)
do not forget to work with the prettier in the last commit